src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -0,0 +1,8 @@
+package wolox.training.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import wolox.training.models.User;
+
+public interface UserRepository  extends CrudRepository<User, Long> {
+    User findByUsername(String username);
```

Comment:
I would use Optional<User> as return type so you can later, if necesarry, use the orElseThrow statement.